src/main/resources/.env

```diff
@@ -0,0 +1,3 @@
+BASE_URL=https://jalasoft-dev-ed.lightning.force.com/lightning/o/Product2/list?filterName=Recent
```

Comment:
you could separate the URL in base URL and the product in other so you could re use them

Reply:
Thanks, I have already fixed that